aspnetcore/fundamentals/host/platform-specific-configuration/samples/2.x/HostingStartupApp/Pages/Error.cshtml

```diff
@@ -19,5 +19,8 @@
     Swapping to the <strong>Development</strong> environment displays detailed information about the error that occurred.
 </p>
 <p>
-    <strong>The Development environment shouldn't be enabled for deployed applications</strong>. It can result in displaying sensitive information from exceptions to end users. For local debugging, enable the Development environment by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong> and restarting the app or adding <code>.UseEnvironment("Development")</code> to <code>WebHost</code> in <i>Program.cs</i> and restarting the app.
+    <strong>The Development environment shouldn't be enabled for deployed applications.</strong>
+    It can result in displaying sensitive information from exceptions to end users.
```

Comment:
Using the Development environment in production can leak sensitive information, for example from exceptions.

Reply:
That's part of the template :point_right: https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Error.cshtml#L23

Reply:
There are 34 additional cases of that line across our topic samples, do you want me to proceed with the change here? If so, I'll patch all 35 instances on this PR.